bin/ghe-restore

```diff
@@ -97,6 +97,14 @@ elif [ "$GHE_VERSION_MAJOR" -ge 2 ]; then
     restore_settings=true
 fi
 
+# Figure out if this instance is in a replication pair
+if ghe-ssh "$GHE_HOSTNAME" -- "ghe-repl-status -r 2>/dev/null" \
+  | grep -Eq "replica|primary"; then
+  instance_configured=true
+  echo "WARNING: Restoring to a server with replication enabled interrupts replication."
+  echo "You will need to reconfigure replication after the restore completes."
```

Comment:
I would indent this second line so it's clear that it is part of the warning. (See example a few lines below).